ext/pdo_sqlite/php_pdo_sqlite_int.h

```diff
@@ -26,28 +26,23 @@ typedef struct {
 	char *errmsg;
 } pdo_sqlite_error_info;
 
-struct pdo_sqlite_fci {
-	zend_fcall_info fci;
-	zend_fcall_info_cache fcc;
-};
-
 struct pdo_sqlite_func {
 	struct pdo_sqlite_func *next;
 
-	zval func, step, fini;
 	int argc;
```

Comment:
Aside: is this `argc` field even used?

Reply:
Yes, here is.
https://github.com/php/php-src/blob/6303d1fc6ad80704e5d57fcb9bd91f557c268b19/ext/pdo_sqlite/sqlite_driver.c#L108